src/main/scala/is/hail/variant/GenomeReference.scala

```diff
@@ -379,6 +380,42 @@ object GenomeReference {
     gr
   }
 
+  def importReferences(hConf: Configuration, path: String) {
+    if (hConf.exists(path)) {
+      val refs = hConf.listStatus(path)
+      refs.foreach { fs =>
+        val grPath = fs.getPath.toString
+        val gr = hConf.readFile(grPath) { (is: InputStream) => fromJSON(JsonMethods.parse(is)) }
```

Comment:
`{ (is: InputStream) => fromJSON(JsonMethods.parse(is)) }` -> `fromJson(JsonMethods.parse(_))`